packages/app/src/AutocompleteInput.tsx

```diff
@@ -38,6 +41,7 @@ export default function AutocompleteInput({
   onLanguageChange?: (language: 'sql' | 'lucene') => void;
   language?: 'sql' | 'lucene';
   showHotkey?: boolean;
+  queryHistoryType?: string;
```

Comment:
Does this,  SearchInputV2.tsx, and SQLInlineEditor.tsx need to have a queryHistoryType prop passed in since they are used depending on the language type anyway?

Reply:
oh, this property just let us easily enable search history in other pages if needed.(ex. alert, session, `queryHistoryType` can make sure they are using different key in localStorage.)